### DISCUSSION TITLE
PCAP/Suricata Logs (Per  Interface?)

### DISCUSSION THREAD

**Discussion by xfaith** [Security-Onion-Solutions/securityonion#7965] [Unsupported Versions] [answered]
Moving from 16.04 --> 2.3 

Are the logs broken out like in 16.04 per interface, or are they all combined under one folder.   

In 16.04 I had multiple "Suricata.yaml" and multiple eve.json (per interface)
Same with PCAP it was broken out per interface

In 2.3 I see  /nsm/PCAP/ and there are files, but does not look like broken out per interface, or named per interface.
Same for Suricata.     

Thanks

**Comment by petiepooo** [comment, marked as answer]
SO 2.3 collects all SPAN interfaces under a single bonded link.  They are all combined.
